packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -79,10 +80,24 @@ export const connectToApp = () => {
   cy.findByText('Connect to your MetaMask Wallet').should('be.visible').click()
 }
 
+export const closeLowBalanceDialog = () => {
```

Comment:
If mainnet account has more than 0.005 ETH the dialog won't appear and tests will fail. This won't be the case in CI but can occur locally.

Reply:
True, we can adjust that when/if we run into that case